compiler/setup.py

```diff
@@ -260,6 +260,8 @@ def prepare_installation():
             "-DCMAKE_BUILD_TYPE={}".format(cfg),
             # TODO(scotttodd): include IREE_TARGET_BACKEND_WEBGPU here (and in env)
             get_env_cmake_option("IREE_ENABLE_CPUINFO", "ON"),
+            get_env_cmake_option("IREE_TARGET_BACKEND_ROCM", "OFF"),
```

Comment:
We are specifically fine with the compiler packages being shipped with rocm enabled. Can you flip this to on

Reply:
done! :)